examples/pettingzoo/meltingpot_env.py

```diff
@@ -0,0 +1,149 @@
+""" PettingZoo interface to meltingpot environments"""
+
+import dm_env
+from meltingpot.python import substrate
+from ml_collections import config_dict
+from gym import spaces
+import numpy as np
+from functools import lru_cache
+
+from gym.utils import EzPickle
+from pettingzoo.utils import wrappers
+from pettingzoo.utils.conversions import parallel_to_aec_wrapper
+from pettingzoo.utils.env import ParallelEnv
+import tree
+import matplotlib.pyplot as plt
+
+PLAYER_STR_FORMAT = 'player_{index}'
+MAX_CYCLES = 1000
+
+def _timestep_to_observations(timestep: dm_env.TimeStep):
+  gym_observations = {}
+  for index, observation in enumerate(timestep.observation):
+    gym_observations[PLAYER_STR_FORMAT.format(index=index)] = {
+        key: value for key, value in observation.items() if 'WORLD' not in key
```

Comment:
Suggest checking if the key _starts_ with 'WORLD.', but if you want to keep it as a `not in` test, maybe at least add the dot? (i.e. 'WORLD.')

Reply:
Yes, I think a lot of duplicated functionality between the RLlib and PettingZoo examples can be moved to utils; however, I would prefer keeping them as is here, and moving them, and adding docstrings and tests in a future PR, if that's okay.